packages/trader/src/App/Components/Animations/slide-in.jsx

```diff
@@ -34,28 +34,38 @@ const SlideInFromBottom = posed.div({
             duration: 100,
         },
     },
-});
+};
 
 const SlideIn = ({ children, className, keyname, is_visible, type }) => {
     if (type === 'bottom') {
         return (
-            <PoseGroup flipMove={false}>
+            <>
```

Comment:
Isn't it changing the behaviour of this component?

Reply:
checked it yesterday. some behaviors didn't work as expected. currently working on them .

Reply:
fixed all issues relating to this component and other ones that use `framer-motion`. thanks @matin-binary  for you revision :pray: